.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    working_directory: ~/workdir
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "build.sbt" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run: 
+          name: Compile
+          command: ./scripts/compile.sh
+
+      - save_cache:
+          key: dependencies-{{ checksum "build.sbt" }}
```

Comment:
This is the old version. Check `https://github.com/codacy/codacy-analysis-cli/blob/fail-cli-with-absolute-values-FT-4987/.circleci/config.yml`